pkg/apis/batch/validation/validation.go

```diff
@@ -568,6 +578,16 @@ func validateJobStatus(job *batch.Job, fldPath *field.Path, opts JobStatusValida
 			}
 		}
 	}
+	if opts.RejectFinishedJobWithTerminatingPods {
+		if status.Terminating != nil && *status.Terminating > 0 && isJobFinished {
+			allErrs = append(allErrs, field.Invalid(fldPath.Child("terminating"), status.Terminating, "terminating>0 is invalid for finished job"))
+		}
+	}
+	if opts.RejectMoreReadyThanActivePods {
+		if status.Ready != nil && *status.Ready > status.Active {
```

Comment:
Is this now possible because terminating pods can't be counted as ready?

Reply:
It is now possible after https://github.com/kubernetes/kubernetes/pull/125546. 

However, even before this PR terminating pods weren't counted as ready. This is because ready pods were counted as a subset of active pods (which exclude terminating), but only active was updated during the syncJob.